packages/react-data-grid/src/cells/headerCells/SortableHeaderCell.js

```diff
@@ -22,13 +23,13 @@ class SortableHeaderCell extends React.Component {
     case null:
     case undefined:
     case DEFINE_SORT.NONE:
-      direction = DEFINE_SORT.ASC;
+      direction = this.props.sortDescendingFirst ? DEFINE_SORT.DESC : DEFINE_SORT.ASC;
       break;
     case DEFINE_SORT.ASC:
-      direction = DEFINE_SORT.DESC;
+      direction = this.props.sortDescendingFirst ? DEFINE_SORT.NONE : DEFINE_SORT.DESC;
```

Comment:
There are no unit tests for `SortableHeaderCell.js` at the moment. If you have time to add some unit tests in `cells/headerCells/__tests__/SortableHeaderCell.spec.js` that would be awesome